doc/howtos/rdtraining/setup.rst

```diff
@@ -0,0 +1,408 @@
+.. _howto/rdtraining/setup:
+
+==============================
+Development Environment Set-up
+==============================
+
+There are multiple ways to install Odoo depending on the intended use case.
+
+This documents attempts to describe the installation option for an internal Odoo R&D developer. We
+assume that you are installing your development environment in an Odoo's standard laptop with Linux
+Mint installed and updated. If you another environment is used, you can refer to :ref:`setup/install/source`
+
+Fetch the sources & configure Git
+=================================
+
+Install and configure git
+-------------------------
+
+.. code-block:: console
+
+    $ sudo apt install git
+    $ git config --global user.name "Your full name"
+    $ git config --global user.email "xyz@odoo.com"
+
+Configure github
+----------------
+
+To fetch the sources and to contribute in the Odoo's development you will need a GitHub user. If you
+don't have already a GitHub user, we will recommend using your trigram (xyz) followed by '-odoo':
+'xyz-odoo'.
+
+
+The easy way to authenticate to GitHub is to use the ssh connection, using the ssh authentication
+will allow you to connect to GitHub without supplying your username or password at each visit.
+
+
+The following instructions are based on the official `GitHub documentation`_.
+
+.. _GitHub documentation: https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
+
+
+In your computer:
+
+
+- Generating a new SSH key, adding it to the ssh-agent and copy the SSH key to your clipboard.
+
+  .. code-block:: console
+
+    $ ssh-keygen -t rsa -b 4096 -C "xyz@odoo.com"
+    $ ssh-add ~/.ssh/id_rsa
+    $ sudo apt-get install xclip
+    $ xclip -sel clip < ~/.ssh/id_rsa.pub
+
+
+In Github:
+
+
+- In the upper-right corner of any page, click your profile photo, then click Settings
+
+  .. image:: https://docs.github.com/assets/images/help/settings/userbar-account-settings.png
+
+- In the user settings sidebar, click SSH and GPG keys.
+
+  .. image:: https://docs.github.com/assets/images/help/settings/settings-sidebar-ssh-keys.png
+
+- Click New SSH key or Add SSH key.
+
+  .. image:: https://docs.github.com/assets/images/help/settings/ssh-add-ssh-key.png
+
+- In the "Title" field, add a descriptive label for the new key.
+- Paste your key into the "Key" field.
+
+  .. image:: https://docs.github.com/assets/images/help/settings/ssh-key-paste.png
+
+- Click Add SSH key.
+
+  .. image:: https://docs.github.com/assets/images/help/settings/ssh-add-key.png
+
+
+Fetch the sources
+-----------------
+
+All the Odoo sources will be located in `/home/$USER/src/`
+
+.. code-block:: console
+
+    $ cd /home/$USER
+    $ mkdir src
+    $ cd src
+    $ git clone git@github.com:odoo/odoo.git
+    $ git clone git@github.com:odoo/enterprise.git
```

Comment:
Maybe it is worth pointing out this takes time... Could we fetch both in parallel? Need to check if it is worth it. By the way, once odoo is fetched, we can suggest to move forward up to "Run the server" and leave the enterprise be fetched in the backgroud.